test-suite/src/function/take.rs

```diff
@@ -0,0 +1,63 @@
+use gluesql_core::executor::EvaluateError;
+
+use {crate::*, gluesql_core::prelude::Value::*};
```

Comment:
In our repo, we recommend to wrap imports by using single `use`, then the cargo fmt can automatically do formatting.